README.md

```diff
@@ -19,6 +19,7 @@ The current list of Contributors and Approvers in the SIGs are listed in the [CO
 | Name      | Responsibilities    |
 | ------------------ | ------------- |
 | [Architecture & Infra](infra) | Defining and maintaining the core ONNX format, the build and CI/CD systems for ONNX repositories, publishing release packages for ONNX, the onnx-docker repository, and creating tools to help integrate with and test against the ONNX standard. This SIG is also the defacto owner of files in the main ONNX repository unless explicitly owned by another SIG. |
+| [Compilers](compilers) | Developing and maintaining core compiler technology optimizing and lowering ONNX format. |
```

Comment:
Did you mean "..core compiler technology _for_ optimizing..."

Reply:
Good suggestion, added